ha/core/event_analyzer/parser/parser.py

```diff
@@ -71,3 +80,44 @@ def parse_event(self, msg: str) -> HealthEvent:
         except Exception as e:
             raise EventAnalyzerError(f"Failed to parse alert. Message: {msg}, Error: {e}")
 
+class IEMParser(Parser):
+    """
+    Parser for IEMs.
+    """
+
+    def parse_event(self, msg: str) -> HealthEvent:
+        """
+        Parse event.
+        Args:
+            msg (str): Msg
+        """
+        try:
+            iem_alert = json.loads(msg)
+
+            # Parse hostname and convert to node id
+            iem_description = iem_alert['sensor_response_type']['info']['description']
+            hostname = re.split("=", re.split(";", re.findall("host=.+", iem_description)[0])[0])[1]
+            key_val = self._confstore.get(f"{const.HOSTNAME_TO_NODEID_KEY}/{hostname}")
```

Comment:
can we add TODO to evaluate mapping as config
and this TODO need to be resolved during mapping task

Reply:
@ajay-paratmandali  Please elaborate more.